modules/ROOT/pages/whatsnew.adoc

```diff
@@ -15,43 +15,15 @@ include::ROOT:partial$_set_page_context.adoc[]
 
 :param-related: {xref-sgw-pg-release-notes}
 include::partial$_show_page_header_block.adoc[]
-// tag::scopes-and-collections[]
 
-CAUTION: {sgw-t} 3.0.x introduces some breaking changes.
-If you are upgrading from 2.x, please refer to the {upgrading--xref} page.
-Users should be able to upgrade to 3.1.x from 3.0.x without manual intervention.
 
-* *New support for Scopes and Collections* +
-Couchbase has introduced support for Scopes and Collections for self-managed cloud-to-edge deployments only in xref:couchbase-lite:ROOT:index.adoc[Couchbase Lite 3.1.0] and xref:introduction.adoc[Sync Gateway 3.1.0]. 
-This release won't cause any issues with existing apps, as it's compatible with older versions. 
-If you have an app that uses bucket-based APIs, you can still upgrade to 3.1, but please note that this API is now deprecated. 
-For more information, see xref:scopes-and-collections-config.adoc[Scopes and Collections Configuration for Sync Gateway].
+* *Audit Logging Support*
 
-* *Improved Data Organization and Access Control for Scopes and Collections* +
-Couchbase Mobile now offers Scopes and Collections, allowing more efficient and scalable data organisation within a bucket. 
-This also introduces an improved method of defining and enforcing data xref:access-control-concepts.adoc[Access Control] more granularly. 
-Multi-tenant apps will also experience better scalability and independent data lifecycle management.
+Couchbase now provides Audit Logging support. Audit Logging provides tools for administrators to track operational irregularities and to support regulatory and security compliance standards, such as link:https://www.hhs.gov/hipaa/index.html[HIPAA] and link:https://soc2.co.uk/soc2[SOC-2].
```

Comment:
Specify that it's audit logging support for Sync Gateway?

Reply:
Done